src/storage/event_listener.cc

```diff
@@ -75,9 +75,20 @@ bool IsDiskQuotaExceeded(const rocksdb::Status &bg_error) {
   return err_msg.find(exceeded_quota_str) != std::string::npos;
 }
 
+void EventListener::OnCompactionBegin(rocksdb::DB *db, const rocksdb::CompactionJobInfo &ci) {
+  LOG(INFO) << "[event_listener/compaction_begin] column family: " << ci.cf_name << ", job_id: " << ci.job_id
+            << ", compaction reason: " << rocksdb::GetCompactionReasonString(ci.compaction_reason)
+            << ", output compression type: " << CompressType2String(ci.compression)
+            << ", base input level(files): " << ci.base_input_level << "(" << ci.input_files.size() << ")"
+            << ", output level(files): " << ci.output_level << "(" << ci.output_files.size() << ")"
+            << ", input bytes: " << ci.stats.total_input_bytes << ", output bytes:" << ci.stats.total_output_bytes
+            << ", is_manual_compaction:" << (ci.stats.is_manual_compaction ? "yes" : "no")
+            << ", elapsed(micro): " << ci.stats.elapsed_micros;
```

Comment:
what would `ci.stats.elapsed_micros` in begin means?

Reply:
That's meaningless for the begin function, so let's remove it.